docs/source/client.rst

```diff
@@ -141,9 +177,9 @@ easy by way of :meth:`.Client.search_range`. For example:
 
 .. ipython:: python
 
-    client.search_range('z', start=314.4, end=348.6, type='Device')
+    client.search_range('z', start=314.4, end=348.6, type='HappiItem')
```

Comment:
I... think it is updated? GitHub isn't showing this in the diff somehow?

Reply:
`client.search_range('z', start=734.0, stop=735.0, device_class='pcdsdevices.attenuator.FEESolidAttenuator')` --> `at2l0` ?

Reply:
I'm all for AT2L0 additions. 🎉 

The only thing is that we need to have something for the docs that exists in happi. That's because these `ipython` sections are actually executed (see the PR body for a screenshot of what these do)

Reply:
Maybe I've been rendered blind, but where does the `docs_test.json` come from?

Reply:
It's a bit weird, but it gets created by those `ipython` blocks in the documentation.

Your comments made me realize that we're actually giving the wrong parameters to those `HappiItem`s and we don't have a good *official* field to even search with `search_range` (though `z` can still work as an extraneous field)

Reply:
Ah I don't mind too much what we do- I just think it's weird for the example to filter on type at all when that's not a thing people ever do. I was trying to advocate for removing the type= here. This is separate from the scope of this PR but the diff brought it to my attention.

Reply:
I made some tweaks to the docs; hopefully they're better than the existing ones